README.md

```diff
@@ -168,6 +168,12 @@ boost_where: {user_id: {value: 1, factor: 100}} # default factor is 1000
 boost_where: {user_id: [{value: 1, factor: 100}, {value: 2, factor: 200}]}
 ```
 
+Boost indices when searching across more than one index
```

Comment:
Can we move this to a section about searching multiple models? (maybe right above the reference section)